api/src/main/java/org/pmiops/workbench/opsgenie/OpsGenieServiceImpl.java

```diff
@@ -0,0 +1,72 @@
+package org.pmiops.workbench.opsgenie;
+
+import com.ifountain.opsgenie.client.swagger.ApiException;
+import com.ifountain.opsgenie.client.swagger.api.AlertApi;
+import com.ifountain.opsgenie.client.swagger.model.CreateAlertRequest;
+import java.util.logging.Logger;
+import javax.inject.Provider;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.model.EgressEvent;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OpsGenieServiceImpl implements OpsGenieService {
+  private static final Logger logger = Logger.getLogger(OpsGenieServiceImpl.class.getName());
+
+  private Provider<AlertApi> alertApiProvider;
+  private Provider<WorkbenchConfig> workbenchConfigProvider;
+
+  @Autowired
+  public OpsGenieServiceImpl(
+      Provider<AlertApi> alertApiProvider, Provider<WorkbenchConfig> workbenchConfigProvider) {
+    this.alertApiProvider = alertApiProvider;
+    this.workbenchConfigProvider = workbenchConfigProvider;
+  }
+
+  private CreateAlertRequest egressEventToOpsGenieAlert(
+      EgressEvent egressEvent, WorkbenchConfig workbenchConfig) {
+    CreateAlertRequest request = new CreateAlertRequest();
+    request.setMessage(String.format("High-egress event (%s)", egressEvent.getProjectName()));
+    request.setDescription(
+        new StringBuilder()
+            .append(String.format("Workspace project: %s\n", egressEvent.getProjectName()))
```

Comment:
I considered this, but it turns out programmatically looking up the username / email / etc is a bit more complicated (effectively, we'd have to extract [this method body](https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/actionaudit/auditors/EgressEventAuditorImpl.kt#L57) into a central service of some sort). I'm not sure the value is huge, since that info is presented directly in the workspace admin landing page which is the clear alert call-to-action.

Reply:
OK. Can we link directly to that page in the description?

Reply:
Yep – I don't disagree it will be nice, but I wanted to hold off from getting too fancy so we have at least something running in prod to start working with.

And see 2 lines below this, we do have a direct link to the admin page.